beetsplug/discogs.py

```diff
@@ -302,6 +303,13 @@ def get_album_info(self, result):
         mediums = [t.medium for t in tracks]
         country = result.data.get('country')
         data_url = result.data.get('uri')
+        style = result.data.get('styles')
+        if style is None:
+            self._log.info('Style not Found')
+        elif len(style) == 0:
```

Comment:
```suggestion
        elif not style:
```

Reply:
This statement is checking to see if there is one item in the array. Is what you suggested, "not style" making the same check?

Reply:
Yes. (In Python, empty lists are falsy.)